src/Microsoft.OpenApi.OData.Reader/Operation/ComplexPropertyPostOperationHandler.cs

```diff
@@ -39,18 +39,17 @@ protected override void Initialize(ODataContext context, ODataPath path)
     /// <inheritdoc/>
     protected override void SetBasicInfo(OpenApiOperation operation)
     {
-        // Summary
-        operation.Summary = $"Sets a new value for the collection of {ComplexPropertySegment.ComplexType.Name}.";
-
         // OperationId
         if (Context.Settings.EnableOperationId)
         {
             string typeName = ComplexPropertySegment.ComplexType.Name;
             operation.OperationId = ComplexPropertySegment.Property.Name + "." + typeName + ".Set" + Utils.UpperFirstChar(typeName);
         }
 
-        // Description
-        operation.Description = InsertRestrictions?.Description;
+        // Summary and Description
+        var placeHolder = $"Sets a new value for the collection of {ComplexPropertySegment.ComplexType.Name}.";
+        operation.Summary = InsertRestrictions?.Description ?? Context.Model.GetDescriptionAnnotation(ComplexPropertySegment.Property) ?? placeHolder;
+        operation.Description = InsertRestrictions?.LongDescription ?? Context.Model.GetLongDescriptionAnnotation(ComplexPropertySegment.Property) ?? placeHolder;
```

Comment:
Do not pick the Long description annotations (`Org.OData.Core.V1.LongDescription`) for CUD operations. The `Org.OData.Core.V1.LongDescription` and `Org.OData.Core.V1.Description` basically just annotate an element by describing it. So it only makes sense for Read operations.

Reply:
Make this change for all the Complex* classes?

Reply:
The idea here was this:-
Where the Green Line becomes Description
Where the Red Lines become LongDescriptions. 

<img width="330" alt="image" src="https://user-images.githubusercontent.com/1641829/161817103-f98b5a74-c986-4ff3-a0e3-36b73c1d0ea0.png">

The Spec seems to support this 
<img width="551" alt="image" src="https://user-images.githubusercontent.com/1641829/161817274-4d3e71f5-f987-435c-b8e1-25c54b422025.png">

Reply:
The `Org.OData.Core.V1.Description`s are different from the `Description`s and `LongDescription`s in the Restrictions annotations. 
e.g., 

![image](https://user-images.githubusercontent.com/40403681/161820441-bdda1a88-ebac-4414-b8a7-cd3a1b14c8c9.png)

or 

![image](https://user-images.githubusercontent.com/40403681/161820812-16535b4f-c18b-486a-a17f-63a72ab33e21.png)

These just describe the particular element, so I think they are usually best left for Read operations. They really won't make sense for other CUD operations.

Reply:
Noted. Making the change.